Mod2/Lessons/Week5/ManyToManyWithAnORM.md

```diff
@@ -0,0 +1,64 @@
+# Many-to-Many With an ORM
+
+## Learning Goals
+*  Understand why frameworks have versions
+*  Reinforce how to structure many-to-many database relationships
+*  Implement a many-to-many relationship using Entity Framework
+
+We're going to continue building our plant tracker application, let's all go ahead and open that app.
+
+### A Bit About Versions
```

Comment:
It feels very fitting to talk about Versions here, but wasn't sure how in depth to go. Definitely interested in your thoughts on if this is too much detail for where they are currently at with the end of mod 2.

Reply:
I agree, this seems like a natural place to introduce versioning.  I think your questions and discussions below are a good intro!